guides/source/asset_pipeline.md

```diff
@@ -1195,3 +1053,51 @@ it as a preprocessor for your MIME type.
 ```ruby
 Sprockets.register_preprocessor 'text/css', AddComment
 ```
+
+
+Alternative Libraries
+------------------------------------------
+
+Over the years there have been multiple default approaches for handling the assets. The web evolved and we started to see more and more Javascript-heavy applications. In The Rails Doctrine we believe that [The Menu Is Omakase](https://rubyonrails.org/doctrine#omakase) so we focused on the default setup: **Sprockets with Import Maps**.
+
+We are aware that there are no one-size-fits-it-all solutions for the various JavaScript and CSS frameworks/extensions available. There are other bundling libraries in the Rails ecosystem that should empower you in the cases where the default setup isn't enough.
+
+### [jsbundling-rails](https://github.com/rails/jsbundling-rails)
+
+A Node.js dependent alternative to the `importmap-rails` way of bundling JavaScript with [esbuild](https://esbuild.github.io/), [rollup.js](https://rollupjs.org/), or [Webpack](https://webpack.js.org/).
+
+The gem provides `yarn build --watch` process to automatically generate output in development. For production, it automatically hooks `javascript:build` task into `assets:precompile` task to ensure that all your package dependencies have been installed and JavaScript has been built for all entry points.
+
+**When to use instead of `importmap-rails`?** If your JavaScript code depends on transpiling so if you are using [Babel](https://babeljs.io/), [TypeScript](https://www.typescriptlang.org/) or React `JSX` format then `jsbundling-rails` is the correct way to go.
+
+### [Webpacker/Shakapacker](https://guides.rubyonrails.org/webpacker.html)
```

Comment:
FYI: while the situations and prospects are still very very unclear, it seems that webpack is being moving to the new Turbopack: 
https://vercel.com/blog/the-turbopack-vision